src/commands/dockersupport/updateContainer.ts

```diff
@@ -0,0 +1,18 @@
+import * as vscode from 'vscode';
+import { AzExtTreeItem, IActionContext } from 'vscode-azureextensionui';
+import { localize } from '../../localize';
+import { getProjectFolder } from './getProjectFolder';
+
+/**
+ * Rebuilds and reopens the opened function app project in a dev container
+*/
+export async function updateContainer(context: IActionContext, node: AzExtTreeItem): Promise<void> {
+
+    try {
+        const projectFilePath: string = await getProjectFolder(context, node.label);
+        vscode.commands.executeCommand('remote-containers.rebuildAndReopenInContainer', vscode.Uri.file(projectFilePath))
+    } catch (error) {
+        const message: string = localize('cannotRebuildandReopen', 'Failed to update and reopen project in a dev container.');
+        throw new Error(message);
```

Comment:
similarly here.. should we throw or just show the error message to the user? also does the `error` object contain any info that could be useful to the user?

Reply:
Nope, the object doesn't contain any info the user needs. Will make the change